views/index.html

```diff
@@ -1,29 +1,204 @@
+                <option value="2">2 - Could be better, but beats working</option>
+                <option value="3">3 - Packed powder, still a solid ride</option>
+                <option value="4">4 - Fresh pow pow, bring your snorkel</option>
+              </select>
+            </div>
+            <!-- end run conditions -->
+            <div class="form-group">
+              <label for="skierAbility">Skier Ability</label>
+              <select class="form-control" id="skierAbility">
+                <option selected disabled>Choose ability</option>
+                <option value="1">Beginner</option>
+                <option value="2">Intermediate</option>
+                <option value="3">Advanced</option>
+                <option value="4">Expert</option>
+                <option value="5">Full send on everything</option>
+              </select>
+            </div>
+            <!-- end skier ability -->
+            <div class="form-group">
+              <label for="runCondition">Notes about run?</label>
+              <textarea class="form-control" id="runCondition" rows="3"></textarea>
+            </div>
+
+            <!-- Button -->
+            <div class="form-group">
+              <label class="col-md-4 control-label" for="singlebutton">Save Ski Run</label>
+              <div class="col-md-4">
+                <input type="submit" value="Send it, brah!" id="singlebutton" name="singlebutton" class="btn btn-primary">
+              </div>
+            </div>
+          </fieldset>
+        </form>
+      </div>
+      </div>
+    </section>
+
+    <section class="container">
+      <div class="row">
+        <div class="col-md-10 col-md-offset-1">
+          <h2>Ski Runs</h2>
         </div>
       </div>
+      <!-- ski runs! -->
+      <div id="ski"></div>
+    </section>
+   
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <h4><a href="/api">Read My API Documentation</a></h4>
+      </div>
     </div>
+
+      <!-- Button trigger modal -->
+      <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#add-ski-run">
+        Add a new ski run
+      </button>
+
+      <!-- Modal -->
+      <div class="modal fade" id="add-ski-run" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title" id="modal-title">Add a new run review</h5>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              <form>
+                <div class="form-group">
+                  <label for="resort">What mountain?</label>
+                  <select class="form-control" id="resort">
```

Comment:
The other reason your frontend isn't working: your inputs don't have `name`s, so they wouldn't show up in your `.serialize()`.